pages/api/checkout/[sessionId].ts

```diff
@@ -1,3 +1,4 @@
+import { ResponseUtil } from "@utils/responseUtil";
 import { NextApiRequest, NextApiResponse } from "next";
 import Stripe from "stripe";
```

Comment:
Should we move Stripe to `services`?

Reply:
Should it be 
```suggestion
import stripe from "@services/stripe";
```

Reply:
good catch - i forgot to change it in this file